fontforge/cvexport.c

```diff
@@ -228,27 +229,24 @@ int _ExportPDF(FILE *pdf,SplineChar *sc,int layer) {
     fprintf( pdf, "    /Creator (FontForge)\n" );
     now = GetTime();
     if (!getenv("SOURCE_DATE_EPOCH")) {
-	tm = localtime(&now);
+	gdt = g_date_time_new_from_unix_local((gint64)now);
```

Comment:
This is missing a `g_date_time_unref` somewhere

Reply:
Eeesh -- And I think of myself as a "programmer". Fixed.